code/src/index.css

```diff
@@ -5,9 +5,159 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  background-image: url("../src/assets/office.jpg");
 }
 
+body, .officeChoice {
+  text-align: initial;
+}
+
+body, .button-finish {
+  text-align: center;
+}
+
+p {
+  font-family:'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  font-size: 24px;
+}
+
+.event-container {
+  font-size: 100px;
+}
+
+.header img {
+  background-size: cover;
+  height: 300px;
+  width: 100%;
+  display: flex;
+  
+ }
+
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.wrapper {
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  background-color: rgba(92, 111, 111, 0.448);
+  margin-top: -200px;
+  height: 300px;
+  width: 600px; 
+} 
+
+.button-next {
+  
+  border: 2px solid #000000cc;
+  width: 110px;
+  height: 30px;
+  
+  
+}
+
+.button-submit {
+  
+  border: 2px solid #000000cc;
+  width: 110px;
+  height: 30px;
+}
+
+.button-next:hover, .button-submit:hover{
+  cursor: pointer;
+  background-color: #65cb8c;
+  
+}
+
+select {
+  background-color: rgba(155, 214, 130, 0.696);
+  color:rgb(43, 34, 34);
+  font-size: 90%;
+  font-weight: 600;
+  border: none; 
+  padding: 5px;
+}
+
+.button-forward {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.button-finish {
+  display: flex;
+}
+
+.eventChoice {
+  display: flex;
+  flex-direction:column;
+}
+
+.officeChoice {
+  font-size: 22px;
+}
+
+input[type="radio" i] {
+  margin: 10px;
+  font-size: x-large;
+}
+
+.startpage {
+display: grid;
```

Comment:
How come you are using grid for the .startpage? I cant see that you are setting the grid anywhere. Since you are not doing that I think it would probably make more sense to use flex och block for display.